ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -237,6 +332,17 @@ public void onProgress(long bytesWritten, long contentLength, boolean done) {
     String contentEncoding = requestHeaders.get(CONTENT_ENCODING_HEADER_NAME);
     requestBuilder.headers(requestHeaders);
 
+    // Check if a handler is registered
+    RequestBodyHandler handler = null;
+    if (data != null) {
+      for (RequestBodyHandler curHandler : mRequestBodyHandlers) {
+        if (curHandler.supports(data)) {
+          handler = curHandler;
```

Comment:
The variable `handler` is never used. A variable was assigned a value that isn't used again.

This whole block looks like it is repeated down in 469. Was that intentional?

Reply:
The blocks are atually slightly different (uri, request and response handlers). Handler should be used tho, something probably got lost in a merge.